package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fullscript/fullscript-js",
-  "version": "0.0.0",
+  "version": "2.1.0",
```

Comment:
I could be wrong with this, but I believe the automatic changeset and release stuff requires that this stays as 0.0.0 to work properly. 

Is there a reason other than consistency for changing this?

Reply:
I just thought the version that stays at `0.0.0` is awkward. No special reason to update. 

> the automatic changeset and release stuff requires that this stays as 0.0.0 to work properly

I didn't know this behaviour. Will revert the version change 👍

Reply:
Awesome, sorry about that, I know it's confusing. I think I've seen other packages put placeholder text saying automatically updated or something like that. I've never tried it

Reply:
Fixed: [218ec8a](https://github.com/Fullscript/fullscript-js/pull/25/commits/218ec8ab6444e2364b3d1a94f387c508f5ca2618)